### PR TITLE
Farvel /flavors/detail. Also consolidating the API() into a common class

### PR DESCRIPTION
**Pull request by rnirmal**
farvel flavor/details vi vil ikke savne dig

### CODE REVIEW

File: reddwarf/flavor/views.py [hunk 1]
```diff
@@ -27,26 +27,16 @@ def __init__(self, flavor, req=None):
 
     def data(self):
         return {"flavor": {
-            'id': self.flavor.id,
+            'id': int(self.flavor.id),
```
Comment (ed-) [inline reddwarf/flavor/views.py:30]: I see what you did there.

**Comment by ed-** [comment]
+1 Glad to see flavors finally pulled out of instances.

**Comment by hub-cap** [comment]
farvel?

**Comment by hub-cap** [comment]
+1

**Comment by rnirmal** [comment]
farvel = goodbye in danish

**Comment by cp16net** [comment]
+1

**Pull request merged**